src/db/models/planVersion.ts

```diff
@@ -37,12 +37,15 @@ export default defineLegacyVersionedModel({
         kind: 'checked',
         type: PLAN_VISIBILITY_PREFERENCES,
       },
+      isPartOfGHO: { kind: 'checked', type: t.boolean },
     },
     required: {
       planId: { kind: 'branded-integer', brand: PLAN_ID },
       name: { kind: 'checked', type: t.string },
+      shortName: { kind: 'checked', type: t.string },
       startDate: { kind: 'checked', type: DATE },
       endDate: { kind: 'checked', type: DATE },
+      planSubtitle: { kind: 'checked', type: t.string },
```

Comment:
[As I wrote](https://github.com/UN-OCHA/hpc_service/pull/3351#discussion_r1733558936) in my review of related PR in hpc_service, let's call it just `subtitle`.